Publish /fr/lecons/gestion-manipulation-donnees-r

anisa-hawes opened the pull request:
Preparing files for publication.

Closes #3429 

### Checklist

- [x] Assign yourself in the "Assignees" menu
- [x] Add the appropriate "Label"
- [x] If this PR closes an Issue, add the phrase `Closes #ISSUENUMBER` to your summary above
- [x] Ensure the status checks pass: if you have difficulty fixing build errors, please contact our Publishing Manager @anisa-hawes 
- [x] Check the Netlify Preview: navigate to netlify/ph-preview/deploy-preview and click 'details' (at right)
- [x] Assign at least one individual or team to "Reviewers"
  - [ ] if the text needs to be translated, please follow the [translation request guidelines](https://github.com/programminghistorian/jekyll/wiki/Requesting-Translation-Guidelines), then assign the relevant language team(s) as "Reviewers" and tag both the team as well as the managing editor in your PR.

anisa-hawes committed:
Upload FR images to data-wrangling-and-management-in-R

For FR translation gestion-manipulation-donnees-r

anisa-hawes committed:
Create gestion-manipulation-donnees-r.md

Create Markdown for new FR translation.

anisa-hawes committed:
Update gestion-manipulation-donnees-r.md

Add translation_date:

marie-flesch commented:
There is just one thing to fix: (level of section heading). 

The section "Et si nous compilions ces fonctions ?" should be H2, or same level as "Qu'est-ce que dplyr?" or "conclusions", like in the English version. 

Otherwise it looks good!

anisa-hawes committed:
Create assets directory /data-wrangling-and-management-in-R

Create /assets/data-wrangling-and-management-in-R

anisa-hawes committed:
Update data-wrangling-and-management-in-R.md

Update /assets link, line 129.

anisa-hawes committed:
Update administracion-de-datos-en-r.md

Update /assets link, line 81.

anisa-hawes committed:
Update gestion-manipulation-donnees-r.md

Update header level, line 461.

anisa-hawes committed:
Move ejemplo_introductorio_estados.csv

Move ES translated asset into directory.

anisa-hawes committed:
Update gestion-manipulation-donnees-r.md

Update /assets link, line 90

anisa-hawes committed:
Rename directory

data-wrangling-and-management-in-R >
data-wrangling-and-management-in-r

anisa-hawes committed:
Update data-wrangling-and-management-in-r.md

data-wrangling-and-management-in-R
>
data-wrangling-and-management-in-r

anisa-hawes committed:
Update manipulacao-transformacao-dados-r.md

manipulacao-transformacao-dados-R
>
manipulacao-transformacao-dados-r

anisa-hawes committed:
Delete images/data-wrangling-and-management-in-R directory

To be replaced with renamed directory:

data-wrangling-and-management-in-R
>
data-wrangling-and-management-in-r

anisa-hawes committed:
Upload /images/data-wrangling-and-management-in-r

Renamed directory: data-wrangling-and-management-in-R > data-wrangling-and-management-in-r

anisa-hawes committed:
Update gestion-manipulation-donnees-r.md

data-wrangling-and-management-in-R > data-wrangling-and-management-in-r throughout

anisa-hawes committed:
Update administracion-de-datos-en-r.md

data-wrangling-and-management-in-R
> 
data-wrangling-and-management-in-r

anisa-hawes committed:
Update manipulacao-transformacao-dados-r.md

Capital R to lowercase r in image filenames.

anisa-hawes committed:
Update data-wrangling-and-management-in-r.md

R > r

anisa-hawes committed:
Rename data-wrangling-and-management-in-r.png

R > r

anisa-hawes committed:
Update beginners-guide-to-twitter-data.md

R > r

anisa-hawes committed:
Update geospatial-data-analysis.md

R > r

anisa-hawes committed:
Update sentiment-analysis-syuzhet.md

R > r

anisa-hawes committed:
Update shiny-leaflet-newspaper-map-tutorial.md

R > r

anisa-hawes committed:
Update analise-sentimento-R-syuzhet.md

R > r

anisa-hawes committed:
Update aplicacao-web-interativa-r-shiny-leaflet.md

R > r

anisa-hawes committed:
Update visualizacao-animacao-tabelas-historicas-R.md

R > r

anisa-hawes committed:
Update aplicacao-web-interativa-r-shiny-leaflet.md

R > r

Commit:
Merge branch 'gh-pages' into publish-gestion-manipulation-donnees-r

anisa-hawes commented:
We look forward to publishing this translation in January ✨

anisa-hawes committed:
Update gestion-manipulation-donnees-r.md

Update `translation_date:`

marie-flesch reviewed (approved):
All good!
